test/suites/lib/tarantool_server.py

```diff
@@ -36,6 +36,26 @@ def find_port(port=None):
         port += 1
     return find_port(3300)
 
+def check_socket(bind_addr):
+    if os.path.exists(bind_addr):
+        return False
+
+    sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+    sock.bind(bind_addr)
+    sock.close()
+
+    return True
+
+def find_socket(bind_num=None):
+    if bind_num is None:
+        bind_num = random.randrange(0, 99)
+    while bind_num < 99:
+        bind_addr = f'/tmp/tarantool_{bind_num}.sock'
```

Comment:
At first glance it will be more portable to use a per session `tmpdir()` from pytest instead of `/tmp/.

https://code-maven.com/pytest-fixtures-temporary-directory
https://github.com/tarantool/tt/blob/master/test/conftest.py

Reply:
Thank you, reworked with `tempfile.NamedTemporaryFile`